CharlieBackend.AdminPanel/HtmlHelpers/CalendarHelper.cs

```diff
@@ -0,0 +1,209 @@
+﻿using CharlieBackend.AdminPanel.Models.Calendar;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Encodings.Web;
+
+namespace CharlieBackend.AdminPanel.HtmlHelpers
+{
+    public static class CalendarHelper
+    {
+        public const int DaysInOneWeek = 7;
+        public static HtmlString CalendarBodyHtml(this IHtmlHelper html, CalendarViewModel calendar)
```

Comment:
Perhaps splitting the code of this long method into smaller private methods would make it more readable?

Reply:
Splitted. 9addd8f7c3c858f432346be32c39219df9043ca0